app/src/main/java/com/kickstarter/libs/utils/KoalaUtils.java

```diff
@@ -226,6 +218,20 @@ private KoalaUtils() {}
     return MapUtils.prefixKeys(properties, prefix);
   }
 
+  public static @NonNull Map<String, Object> refTagProperties(final @Nullable RefTag intentRefTag, final @Nullable RefTag cookieRefTag) {
+    return new HashMap<String, Object>() {
+      {
```

Comment:
Do you need this extra bracket?

Reply:
Yes! It's the `init` block of the `HashMap`